src/fides/api/service/connectors/query_configs/bigquery_query_config.py

```diff
@@ -227,3 +227,45 @@ def format_fields_for_query(
             else:
                 formatted_fields.append(field_path.levels[0])
         return formatted_fields
+
+    def generate_raw_query(
+        self, field_list: List[str], filters: Dict[str, List[Any]]
+    ) -> Optional[TextClause]:
+        """
+        Allows executing a somewhat raw query where the field_list and filters do not depend
+        on the Node or Graph structure.
+
+        This is an override of the base class method that supports nested fields for BigQuery.
+        """
+        clauses = []
+        query_data = {}
+        for field_name, field_value in filters.items():
+            # Replace dots with underscores in field names for parameter binding
+            field_binding_name = field_name.replace(".", "_")
```

Comment:
Key change - This allows us to map dot-delimited `input_data` keys without the query builder treating them as objects